app/src/editor/canvas/components/Resizable/index.jsx

```diff
@@ -121,6 +126,36 @@ const Resizable = ({
         })
     }, [width, height])
 
+    const updatePreviousSize = useCallback(() => {
+        previousSize.current = {
+            height,
+            width,
+        }
+    }, [width, height])
+
+    const updatePreviousSizeRef: Ref<Function> = useRef(updatePreviousSize)
+
+    useEffect(() => {
+        updatePreviousSizeRef.current = updatePreviousSize
+    }, [updatePreviousSize])
+
+    const commitRef: Ref<Function> = useRef(commit)
+
+    useEffect(() => {
+        commitRef.current = commit
+    }, [commit])
+
+    useEffect(() => {
+        const commit: (any) => void = (commitRef.current: any)
+        const updatePreviousSize: () => void = (updatePreviousSizeRef.current: any)
+
+        updatePreviousSize()
+        commit({
+            dx: 0,
+            dy: 0,
+        })
+    }, [minWidth, minHeight])
```

Comment:
The reason for all them refs to be here (`commitRef` and `updatePreviousSizeRef`) is that this effect should only run when `minWidth` or `minHeight` changes, FYI.

Reply:
I don't really understand what's happening here :D Is the `updatePreviousSize` redefined here is it still using it form the upper scope?

Reply:
@juhah so `commitRef` stores a reference to the current value of `commit`. @mondoreale wants to call `commit`, but doesn't want to add `commit` to the list of dependencies for this Effect because `commit` is re-defined whenever `updateSize, onResize, width, height` change, but this effect "should only run when `minWidth` or `minHeight` changes".

Reply:
@juhah so @mondoreale could use the `commit` from the outer scope and leave out `commit` from the list of dependencies, and I *think* that would just work, but the `react-hooks/exhaustive-deps` eslint rule won't be happy about it. I think it's safer to keep the rule enabled and use a ref.

Reply:
Yep, I see it now... The combination of hooks magic and the flow typing makes this a bit hard to read.

Reply:
Hooks give u a lot of flexibility, way more than class components, but it may become confusing. I added a tiny comment about them refs and applied what @timoxley suggested in https://github.com/streamr-dev/streamr-platform/pull/509#pullrequestreview-245253531.